src/game/gameTimer.js

```diff
@@ -1,11 +1,8 @@
-import { finishGame } from "./gameResult";
-
 // container class where will be displayed timer. Change classname if it is necessary
-const TIMER = document.querySelector(".timer");
-
 let gameTime;
 
-export function setTimer(time = 10) {
+function setTimer(time = 10, cb) {
```

Comment:
In the case of `callback` - `cb` s overall good enough, but for fire try to choose more verbose names for passed variables that make it obvious for everyone and always, what It is.

Reply:
called `finishGame`